test/pls.jl

```diff
@@ -35,12 +37,19 @@ const fms = Dict(
     ],
 )
 
-fits = Dict(k => fit.(MixedModel, fms[k], Ref(MixedModels.dataset(k))) for k in keys(fms))
+const fittedmodels = Dict{Symbol,Vector{LinearMixedModel}}();
+function models(nm::Symbol)
+    if haskey(fittedmodels, nm)
+        fittedmodels[nm]
+    else
+        fittedmodels[nm] = fit.(MixedModel, fms[nm], Ref(dataset(nm)))
+    end
+end
```

Comment:
This somehow makes me very happy, as does the commit message.

Reply:
There is also a special method for `get!` that allows you to use a thunk so the method could be written
```julia
models(nm::Symbol) = get!(fittedmodels, nm) do nm
     fit.(MixedModel, fms[nm], Ref(dataset(nm))
end
```
except that the "thunk" function cannot take an argument.  Thus this, slightly wordier, method.